appveyor.yml

```diff
@@ -35,7 +35,7 @@ before_build:
        }
 - ps:  Start-Process clcache-server
 build_script:
-- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nowarn:C4244;C4267;C4715 /nologo
+- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nowarn:C4244;C4267;C4715;C4805 /nologo
```

Comment:
I think instead of bundling that in /nowarn we should include the nowarn inside the generated project file.

Reply:
`<PropertyGroup><NoWarn>C4244;C4267;C4715;C4805</NoWarn></PropertyGroup>`

Reply:
@NicolasDorier I'll fix that! I'm not really familiar with the MSVC project files - in which of them should that be included?

These are the files with `PropertyGroup`:

```
$ git grep -l '<PropertyGroup'
build_msvc/bench_bitcoin/bench_bitcoin.vcxproj
build_msvc/bitcoin-cli/bitcoin-cli.vcxproj
build_msvc/bitcoin-tx/bitcoin-tx.vcxproj
build_msvc/bitcoind/bitcoind.vcxproj
build_msvc/common.init.vcxproj
build_msvc/common.vcxproj
build_msvc/libbitcoin_cli/libbitcoin_cli.vcxproj.in
build_msvc/libbitcoin_common/libbitcoin_common.vcxproj.in
build_msvc/libbitcoin_crypto/libbitcoin_crypto.vcxproj.in
build_msvc/libbitcoin_qt/libbitcoin_qt.vcxproj
build_msvc/libbitcoin_server/libbitcoin_server.vcxproj.in
build_msvc/libbitcoin_util/libbitcoin_util.vcxproj.in
build_msvc/libbitcoin_wallet/libbitcoin_wallet.vcxproj.in
build_msvc/libbitcoin_zmq/libbitcoin_zmq.vcxproj.in
build_msvc/libbitcoinconsensus/libbitcoinconsensus.vcxproj
build_msvc/libunivalue/libunivalue.vcxproj
build_msvc/test_bitcoin-qt/test_bitcoin-qt.vcxproj
build_msvc/test_bitcoin/test_bitcoin.vcxproj
build_msvc/testconsensus/testconsensus.vcxproj
```

Reply:
Tried this but it didn't suppress the warnings:

```diff
diff --git a/appveyor.yml b/appveyor.yml
index 147f458d9..ae6bd2f89 100644
--- a/appveyor.yml
+++ b/appveyor.yml
@@ -35,7 +35,7 @@ before_build:
        }
 - ps:  Start-Process clcache-server
 build_script:
-- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nowarn:C4244;C4267;C4715;C4805 /nologo
+- cmd: msbuild /p:TrackFileAccess=false /p:CLToolExe=clcache.exe build_msvc\bitcoin.sln /m /v:q /nologo
 after_build:
 - cmd: move build_msvc\%PLATFORM%\%CONFIGURATION%\*.iobj build_msvc\cache\
 - cmd: move build_msvc\%PLATFORM%\%CONFIGURATION%\*.ipdb build_msvc\cache\
diff --git a/build_msvc/common.init.vcxproj b/build_msvc/common.init.vcxproj
index c3c0f665c..4ba2aca6c 100644
--- a/build_msvc/common.vcxproj
+++ b/build_msvc/common.vcxproj
@@ -7,9 +7,12 @@
           $(BuildDependsOn);
       </BuildDependsOn>
   </PropertyGroup>
+  <PropertyGroup>
+      <NoWarn>C4244;C4267;C4715;C4805</NoWarn>
+  </PropertyGroup>
   <Target Name="CopyConfig"
           Inputs="$(MSBuildThisFileDirectory)bitcoin_config.h"
           Outputs="$(MSBuildThisFileDirectory)..\src\config\bitcoin-config.h">
       <Copy SourceFiles="$(MSBuildThisFileDirectory)bitcoin_config.h" DestinationFiles="$(MSBuildThisFileDirectory)..\src\config\bitcoin-config.h" />
   </Target>
```

Reply:
You normally did right. I take a look.